uis/layover/app/js/app.js

```diff
@@ -57,6 +57,8 @@ indy.config(['$routeProvider', '$controllerProvider', '$compileProvider', '$filt
   $routeProvider.when('/nfc/view/all', {templateUrl: 'partials/nfc.html'});
   $routeProvider.when('/nfc/view/:packageType/:type/:name', {templateUrl: 'partials/nfc.html'});
 
+  $routeProvider.when('/delete-cache', {templateUrl: 'partials/delete-cache.html'});
```

Comment:
Can we rename the route to `/cache/delete` so we have room for other cache operations someday?

Reply:
Yeah, /cache/delete is really RESTful.